lib/cli.js

```diff
@@ -12,9 +12,14 @@ var meow = require('meow');
 var list = require('cli-list');
 var pkg = require('../package.json');
 var Router = require('./router');
-
 var gens = list(process.argv.slice(2));
 
+/* eslint new-cap: 0, no-extra-parens: 0 */
+var tabtab = new (require('tabtab').Commands.default)({
```

Comment:
Just do this with tabtab: https://github.com/sindresorhus/pageres/commit/5196bd630b4c6427fd9c7fe6266bec68783e2ac8

-

Also do the `require` at the top with the others.